README.md

```diff
@@ -41,14 +45,6 @@ grunt serve
 ```
 
 The project is also accessible at http://localhost:5000
- 
-To package the application into an OpenFin exe run the task
-
-```
-grunt createZip
-```
-
-This will create a zip file for the OpenFinD3FC.exe, which can then be run locally in production.
```

Comment:
This was removed due to it not being relevant any more (create zip creates a zip for the Github Pages url)